docs/releases/1.10.txt

```diff
@@ -890,6 +890,11 @@ Miscellaneous
 * If you subclass ``AbstractBaseUser`` and override ``clean()``, be sure it
   calls ``super()``. :meth:`.AbstractBaseUser.normalize_username` is called in
   a new :meth:`.AbstractBaseUser.clean` method.
+* If you subclass ``AbstractUser`` and override ``clean()``, be sure it calls
```

Comment:
Please move this to the 1.11 release notes and also update the versionadded annotation.